docs/source/user_guide/faq.rst

```diff
@@ -30,6 +30,33 @@ Should I use PySpark's DataFrame API or Koalas?
 If you are already familiar with pandas and want to leverage Spark for big data, we recommend
 using Koalas. If you are learning Spark from ground up, we recommend you start with PySpark's API.
 
+Can I use Koalas with Jupyter and PySpark?
```

Comment:
Hmmm .. the guide looks like more focused on how to install PySpark rather than Koalas. In addition in [Binder integration](https://mybinder.org/v2/gh/databricks/koalas/master?filepath=docs%2Fsource%2Fgetting_started%2F10min.ipynb) (which is a live Jupyter notebook), I merely [install pyspark alone](https://github.com/databricks/koalas/blob/master/postBuild#L4).

Reply:
I like the binder integration but takes a while for it to launch and setup, plus it's good for a quick small, sample Jupyter notebooks we provide as part of 10min if they want to peruse interactively without writing them.

However, if they want to write their own Jupyter notebooks and have already followed the instructions on how to install Koalas, along with PySpark, the aforementioned two methods show them how to use Koalas and PySpark on Jupyter.

Reply:
I also think the propositions in my question are misplaced. It should read:
"Can I use Koalas and PySpark in Juypter?"